.jules/runs/compat_run_1/pr_body.md

```
@@ -0,0 +1,50 @@
+## 💡 Summary
+Replaced unguarded usages of `tokmd_git::git_cmd()` with `std::process::Command::new("git")` to fix compilation failures when building `tokmd` with `--no-default-features`. Additionally, silenced dead code compiler warnings in `export_bundle.rs` that appeared when the `analysis` feature was disabled.
+
+## 🎯 Why
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)